nautobot/ipam/tables.py

```diff
@@ -264,15 +262,15 @@ class Meta(BaseTable.Meta):
 class VRFDeviceAssignmentTable(BaseTable):
     """Table for displaying VRF Device Assignments with RD."""
 
-    vrf = tables.LinkColumn(verbose_name="VRF")
-    device = tables.LinkColumn()
+    vrf = tables.Column(verbose_name="VRF", linkify=lambda record: record.vrf.get_absolute_url(), accessor="vrf.name")
```

Comment:
Might be nice to have namespace shown here too. The screenshot of this table has VRFs from two namespaces but it's not obvious which VRF belongs to which namespace. Perhaps use `accessor="vrf.display"`? (`f"{self.namespace}: ({self.name})"`)

![image](https://user-images.githubusercontent.com/75227981/229157652-0ef4ffdb-39d9-4224-87fe-5a2306f670a2.png)

Reply:
Also, would it make more sense for `VRF.display` to be `f"{self.name} ({self.namespace})"`